modules/ipi-install-verifying-static-ip-address-configuration.adoc

```diff
@@ -0,0 +1,20 @@
+// This is included in the following assemblies:
+//
+// ipi-install-installation-workflow.adoc
+
+[id='verifying-static-ip-address-configuration_{context}']
+
+= Verifying static IP address configuration
+
+If the DHCP reservation for a cluster node specifies an infinite lease, after the installer successfully provisions the node, the dispatcher script will check the node's network configuration. If the script determines that the network configuration contains an infinite DHCP lease, it creates a new connection using the IP address of the DHCP lease as a static IP address.
+
+[NOTE]
+====
+The dispatcher script may run on successfully provisioned nodes while the provisioning of other nodes in the cluster is ongoing.
+====
+
+To verify the network configuration is working properly, you can:
```

Comment:
Per IBM SG, try to avoid trailing off into a semi-colon/list mid-sentence. Could this also be reworked as a `.Procedure`?